apps/ex_wire/lib/ex_wire/rank/connection_observer.ex

```diff
@@ -0,0 +1,128 @@
+defmodule ExWire.Rank.ConnectionObserver do
```

Comment:
maybe we can add this module under `Kademlia` namespace?
(`ExWire.Kademlia.Rank.ConnectionObserver`)

Reply:
Yeah makes sense!

Reply:
@InoMurko Actually, it was a bad idea. Because this module keeps track of not kademlia nodes but peers

Reply:
Yeah - I thought it’s ok since the peers come from the same protocol. But 🤘👍🏻